ibis/pandas/execution.py

```diff
@@ -872,3 +913,175 @@ def execute_array_collect(op, data, **kwargs):
 @execute_node.register(ops.ArrayCollect, SeriesGroupBy)
 def execute_array_collect_group_by(op, data, **kwargs):
     return data.apply(list)
+
+
+@execute_node.register(ops.CumulativeSum, pd.Series)
+def execute_series_cumsum(op, data, **kwargs):
+    return data.cumsum()
+
+
+@execute_node.register(ops.CumulativeMin, pd.Series)
+def execute_series_cummin(op, data, **kwargs):
+    return data.cummin()
+
+
+@execute_node.register(ops.CumulativeMax, pd.Series)
+def execute_series_cummax(op, data, **kwargs):
+    return data.cummax()
```

Comment:
Need to add a test to make sure these do or don't work on `SeriesGroupBy` as well.